tests/conditions.py

```diff
@@ -7,7 +7,7 @@
 has_crashpad = sys.platform != "linux" and not is_android
 # 32-bit linux has no proper curl support
 has_http = not is_android and not is_x86
-has_inproc = sys.platform != "win32"
+has_inproc = true
```

Comment:
```suggestion
has_inproc = True
```

Oh well, I made that same mistake a couple of times. If you want to take it further, just remove the var and all its uses. Just grep in the codebase for this.

Reply:
Hmm I don't have a "format" target 

```
[build@97a1ca2f4064 build-linux64]# make format
make: *** No rule to make target `format'.  Stop.
```

Reply:
https://github.com/getsentry/sentry-native/blob/f91ed3f95b5653f247189d720ab00765b4899d6f/Makefile#L64-L76 well it is there. Maybe you tried to run it in the build directory?

Reply:
Ahh.. yup....

I'll have to figure out what's needed to get that running, get lots of python errors running it.

Reply:
meh, in that case just run the clang-format command directly. I can also take the python changes from here.

Reply:
code all reformatted.